src/components/ArticlePage.jsx

```diff
@@ -18,6 +20,18 @@ const ArticlePage = () => {
         console.error("Error fetching article:", error);
         setError("Failed to fetch article. Please try again later.");
       });
+
+    axios
+      .get(
+        `https://be-nc-news-r6yn.onrender.com/api/articles/${articleId}/comments`
+      )
```

Comment:
Ideally your requests to the api would be in their own files to help separate your concerns a bit - usually we'd call this `api.js`

Having your requests in their own self-contained functions helps keep your component files clean!